docs/src/makie.md

```diff
@@ -0,0 +1,79 @@
+# Makie.jl plots
+
+This page shows an example of plotting MCMCChains.jl with Makie.jl.
+The example is meant to provide an useful basis to build upon.
+Let's define some random chain and load the required packages:
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+A basic way to visualize the chains is to show the drawn samples at each iteration.
+Colors depict different chains:
+
+```@example makie
+params = names(chns, :parameters)
+
+df = DataFrame(chns)
+n_chains = length(unique(df.chain))
+n_samples = nrow(df) / n_chains
+
+# Alternatively, use `CategoricalArrays.categorical`.
+df[!, :chain] = string.(df.chain)
+
+fig = Figure(; resolution=(1_000, 800))
+
+# Create and store separate axes for showing iterations.
+values_axs = [Axis(fig[i, 1]; ylabel=string(c)) for (i, c) in enumerate(params)]
+for (ax, col) in zip(values_axs, params)
+    for i in 1:n_chains
+        chain = string(i)
+        values = filter(:chain => ==(chain), df)[:, col]
+        lines!(ax, 1:n_samples, values; label=chain)
+    end
+end
+
+# Thanks to having stored the axes before, we can apply some extra tweaks.
+# These tweaks usually depend on the kind of model being fitted.
+values_axs[end].xlabel = "Iteration"
+hidexdecorations!.(values_axs[1:end-1]; grid=false)
+
+fig
```

Comment:
Can we ouput SVGs?

Reply:
Yes. Changed.

Doesn't matter much for file size though:
```
-rw-r--r-- 1 rik users 255K Dec 25 10:55 index.png
-rw-r--r-- 1 rik users 233K Dec 25 10:54 index.svg
```

Reply:
I'm not concerned about file size but the quality of the plots.